selfdrive/car/body/fingerprints.py

```diff
@@ -16,7 +16,6 @@
       b'02/27/2022',
       b'0.3.00a',
     ],
-    # git hash of the firmware used
```

Comment:
something like this would be tricky to keep because we'd have to parse the python ourselves or maybe use something like AST

Reply:
just high level comments are fine